eventually/amazons3/views.py

```diff
@@ -44,15 +45,15 @@ def post(self, request):
         """
 
         if request.content_type != 'multipart/form-data':
-            return HttpResponse(status=400)
+            return RESPONSE_400_INVALID_DATA
```

Comment:
maybe we could add addition response? something like `RESPONSE_400_IVALID_HTTP_HEADER` how do you think?

Reply:
No need for it now, as this part of code has been moved to the external module which doesn't return http response but a boolean value.